packages/odyssey-react-mui/src/theme/mixins.ts

```diff
@@ -10,12 +10,19 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import type { ThemeOptions } from "@mui/material";
+import type { MixinsOptions } from "@mui/material";
 import * as Tokens from "@okta/odyssey-design-tokens";
+import { TokenOverrideOptions } from ".";
+import { deepmerge } from "@mui/utils";
 
-export const mixins: ThemeOptions["mixins"] = {
-  maxWidth: Tokens.FontLineLengthMax,
-  borderRadius: Tokens.BorderRadiusBase,
-  borderStyle: Tokens.BorderStyleBase,
-  borderWidth: Tokens.BorderWidthBase,
+export const mixins = (
+  tokenOverrides?: TokenOverrideOptions
+): MixinsOptions => {
+  const odysseyTokens = deepmerge(Tokens, tokenOverrides);
+  return {
+    maxWidth: odysseyTokens.FontLineLengthMax,
+    borderRadius: odysseyTokens.BorderRadiusBase,
+    borderStyle: odysseyTokens.BorderStyleBase,
+    borderWidth: odysseyTokens.BorderWidthBase,
+  };
 };
```

Comment:
I'm not understanding why we're doing a deep merge here.

Odyssey tokens are flat, so a shallow `{ ...tokenOverrides }` should be enough, and this merging can happen in `createOdysseyTheme`; it doesn't have to happen in each theme object right?

I'd rather we pass tokens to each function, not the overrides:
```suggestion
export const mixins = (
  odysseyTokens: OdysseyTokens
): MixinsOptions => ({
  maxWidth: odysseyTokens.FontLineLengthMax,
  borderRadius: odysseyTokens.BorderRadiusBase,
  borderStyle: odysseyTokens.BorderStyleBase,
  borderWidth: odysseyTokens.BorderWidthBase,
});
```

Reply:
The only reason for `deepmerge` was that it would continue to work even if we decide to introduce nested objects inside the Tokens later on.